.github/workflows/end-to-end-testing.yml

```diff
@@ -8,9 +8,15 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-      - name: Run Cypress 🌲
+      - name: Run Cypress for the Dataset Browser 🌲
         uses: cypress-io/github-action@v5
         with:
           project: ./apps/dataset-browser
+        env:
+          CYPRESS_BASE_URL: ${{ github.event.deployment_status.target_url }}
+      - name: Run Cypress for the Research app 🌲
+        uses: cypress-io/github-action@v5
+        with:
+          project: ./apps/research
         env:
```

Comment:
@sdevalk I don't think I set the `CYPRESS_BASE_URL` correctly here. Would you happen to know how I can set this?